cupy/_logic/content.py

```diff
@@ -39,7 +41,57 @@ def _create_float_test_ufunc(name, doc):
     ''')
 
 
-# TODO(okuta): Implement isneginf
-
-
-# TODO(okuta): Implement isposinf
+def isneginf(x, out=None):
+    """Returns a bool array, where True if input element is negative infinity.
+    Args:
+        x (cupy.ndarray): Input array.
+    Returns:
+        cupy.ndarray: Boolean array of same shape as ``x``.
+    Examples
+    --------
+    >>> cupy.isneginf(0)
+    False
+    >>> cupy.isneginf([4, -4, numpy.inf, -numpy.inf])
+    [False, False, False, True]
+
+    .. seealso:: :func:`numpy.isneginf`
+
+    """
+
+    is_inf = cupy.isinf(x)
+    try:
+        signbit = cupy.signbit(x)
+    except TypeError as e:
+        dtype = cupy.asanyarray(x).dtype
+        raise TypeError(f'This operation is not supported for {dtype} values '
+                        'as it would be uncertain.') from e
```

Comment:
Please exactly copy the numpy's exception message unless it is necessary to change.